dataverse-webapp/src/main/webapp/resources/js/fileupload.js

```diff
@@ -39,8 +39,9 @@ function uploadStarted() {
 	observer.observe(files[0].parentElement,config);
 }
 
-function uploadFinished(fileupload) {
-    if (fileupload.files.length === 0) {
+function uploadFinished() {
+    var files = $('.ui-fileupload-files .ui-fileupload-row');
+    if (files.length === 1) {
```

Comment:
Shouldn't this be zero-length on removing last file?

Reply:
In fact we have one additional element on that list, so it's never empty.